backend/app/Http/Resources/v1/CategoryCollection.php

```diff
@@ -23,6 +23,6 @@ class CategoryCollection extends ResourceCollection
     public function toArray($request)
     {
 
-        return $this->collection->map->only(['id', 'name', 'builtin']);
+        return $this->collection->map->only(['id', 'name', 'builtin', 'entities']);
```

Comment:
No need to map through everything, since those are the only 4 fields available anyway.

```suggestion
        return $this->collection;
```